build/package/windows/nri-386-installer/nri-installer.wixproj

```diff
@@ -44,9 +44,9 @@
         </CreateProperty>
     </Target>
     <Target Name="SignInstaller">
-        <Exec Command="&quot;$(SignToolPath)signtool.exe&quot; sign /s &quot;My&quot; /d &quot;$(pfx_certificate_description)&quot; /n &quot;$(pfx_certificate_description)&quot; &quot;$(OutputPath)$(OutputName).msi&quot;"/>
+        <Exec Condition="'$(noSign)' != 'true'" Command="&quot;$(SignToolPath)signtool.exe&quot; sign /s &quot;My&quot; /d &quot;$(pfx_certificate_description)&quot; /n &quot;$(pfx_certificate_description)&quot; &quot;$(OutputPath)$(OutputName).msi&quot;"/>
```

Comment:
i guess that this is for backward compatibility but otherwise I would change it to a non-inverted flag like `sign==true`

Reply:
By default it signs everything, changing it to `SIGN == true` imply to change add a variable to everything and set up a default while this is only for the `fake-prerelease` target.

I would prefer to leave it as it is right now.